src/backend/src/common/entities/base.entity.ts

```diff
@@ -38,4 +38,19 @@ export class BaseEntity {
     name: 'created_by_username',
   })
   createdByUsername: string;
+
+  @Column({
+    name: 'updated_by_guid',
+  })
+  updatedByGuid: string;
+
+  @Column({
+    name: 'updated_by_username',
+  })
+  updatedByUsername: string;
+
+  @Column({
+    name: 'save_id',
```

Comment:
maybe named as saved_id so it can consistent with updated_by_*

Reply:
POV: save_id sounds better to me